app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,283 @@
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>const promise = account.updatePhoneSession('[USER_ID]', '[SECRET]');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    If the provided user ID has not been registered, a new account will be created.
+</p>
+
+<h2><a href="/docs/authentication#login" id="login">Login</a></h2>
+
+<p>
+    Logging in with an email and password is one of the most common ways to login into an application. 
+</p>
+
+<p>
+    The example below shows you how to create a session using the Web SDK:
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.createEmailSession('email@example.com', 'password');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    When a user tries to access restricted resources, you can check if they have a valid, active session. The Account Service provides a <a href="/docs/account#accountGet">get()</a> method that checks whether the current user session is active and returns the account information if successful.
+</p>
+
+<p>
+    The example below shows you how to check whether there is an active session using the Web SDK:
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.getSession('current');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    An authenticated session in appwrite lasts for 1 year and is then automatically destroyed.
+</p>
+
+<h2><a href="/docs/authentication#persistence" id="persistence">Persistence</a></h2>
+
+<p>
+    Appwrite handles the persistence of the session in a consistent way across SDKs. After authenticating with an SDK, the SDK will persist the session so that the user will not need to log in again the next time they open the app. How the session is persisted depends on the SDK.
```

Comment:
```suggestion
    Appwrite handles the persistence of the session in a consistent way across SDKs. After authenticating with an SDK, the SDK will persist the session so that the user will not need to log in again the next time they open the app. The mechanism for persistence depends on the SDK.
```